src/main/java/bio/terra/cli/serialization/userfacing/input/controlled/CreateBqDatasetParams.java

```diff
@@ -1,5 +1,6 @@
-package bio.terra.cli.serialization.userfacing.input;
+package bio.terra.cli.serialization.userfacing.input.controlled;
```

Comment:
This package reorganization means we now have 2 indications about whether a class pertains to a controlled/referenced resource: the package name (`input.controlled` or `input.referenced`) and the class name (`Create...Params` or `Add...Params`). I don't think this repetition is useful. Either we should differentiate by class name, as it already is I think, or reorganize into separate packages and remove the class name prefixes.